plugins/git/plugin.zsh

```diff
@@ -29,6 +29,7 @@ PROMPT_GEOMETRY_GIT_TIME_SHOW_EMPTY=${PROMPT_GEOMETRY_GIT_TIME_SHOW_EMPTY:-true}
 
 # Misc configurations
 GEOMETRY_GIT_NO_COMMITS_MESSAGE=${GEOMETRY_GIT_NO_COMMITS_MESSAGE:-"no commits"}
+GEOMETRY_GIT_SEPARATOR=${GEOMETRY_GIT_SEPARATOR:-"::"}
```

Comment:
Technically there is no problem to implement it. My only concern is that it will look a bit cluttered IMO. I would add a config for plugin separators `GEOMETRY_PLUGIN_SEPARATOR` and then leave the option to the user to set both to the same value.